demo/src/App.vue

```diff
@@ -219,10 +261,53 @@ export default {
     },
     backgroundColor() {
       return this.chosenColor === 'dark' ? this.colors.messageList.bg : '#fff'
+    },
+    messageList() {
+      const chat = this.chatHistory[this.currentChatID];
+      if (chat === undefined) {
+        return [];
+      }
+      return chat["messages"];
+    },
+    participants() {
+      const chat = this.chatHistory[this.currentChatID];
+      if (chat === undefined) {
+        return [];
+      }
+      return chat["participants"];
```

Comment:
Parts of this diff has semi-colons. Can you please run `npm run lint` and update this PR? It should fix most style issues automatically.